tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/PluginPackageResourceListener.java

```diff
@@ -413,6 +413,13 @@ public IStatus runInWorkspace( IProgressMonitor monitor ) throws CoreException
 
     protected void processPropertiesFile( IFile pluginPackagePropertiesFile ) throws CoreException
     {
+        if ( !pluginPackagePropertiesFile.exists() )
+        {
+            ProjectCore.logError( ProjectCore.createErrorStatus( pluginPackagePropertiesFile + "does not exists." ) );
+
```

Comment:
hey @gamerson , @simonjhy 
I think we should be very careful to use **logError** and **Exception** right now, if we use **logError** that means the **IDE** has problem and it will be catch by Error Report to make user think **IDE** got wrong.
For this case, I don't think we need to log error anything and just return will be OK. if we want to mark status we should make return value as Status and return an Error status and get this value from the caller:
@Override
public IStatus runInWorkspace( IProgressMonitor monitor ) throws CoreException
{
    final IResource resource = delta.getResource();

    processPropertiesFile( (IFile) resource );

    return Status.OK_STATUS;
}

Reply:
Agreed